src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable no-use-before-define */
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import {FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
 import {useNavigation, useRoute} from '@react-navigation/native';
 
-import PropTypes from 'prop-types';
 import DiscoveryAction from '../../../context/actions/discoveryAction';
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed, but it doesn't seem to affect the functionality of the code. However, it's a good practice to keep related imports together for better readability and maintainability. In this case, all the imports from 'react-native' should be grouped together.
```diff
- import * as React from 'react';
- import PropTypes from 'prop-types';
- import {FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
- import {useNavigation, useRoute} from '@react-navigation/native';
+ import * as React from 'react';
+ import {FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
+ import {useNavigation, useRoute} from '@react-navigation/native';
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->